api/src/tasks/generateStatsQueue.ts

```diff
@@ -0,0 +1,36 @@
+import { Worker, Queue, QueueScheduler } from "bullmq";
+import createDebug from "debug";
+import Redis from "ioredis";
+import { generateDataStats } from "../index";
+import { config } from "../config";
+
+const queueName = "generate-data-stats";
+const debug = createDebug(`queue:${queueName}`);
+const redisClient = new Redis(config.REDIS_URL);
+
+export const generateDataStatsQueueScheduler = new QueueScheduler(queueName, {
+  connection: redisClient,
+});
+export const generateDataStatsQueue = new Queue<{}>(queueName, {
+  connection: redisClient,
+});
+
+const worker = new Worker(
+  queueName,
+  async () => {
+    await generateDataStats()
+      .then(() => {
+        debug(`Successfully generated data`);
```

Comment:
this part should still contain the call to the rebuild url otherwise the frontend won't be updated